app/__tests__/fixtures/rpc-responses/common-base-instruction-card-getAddressLookupTable-cuo2mz.json

```diff
@@ -0,0 +1,1010 @@
+{
```

Comment:
Is there a reson we write the type for each value into fixture? I think that it is more reasonable to keep it as close to API response as possible and convert into proper type upon reading the fixture